src/main/groovy/org/jenkinsci/plugins/sshsteps/SSHService.groovy

```diff
@@ -103,14 +110,26 @@ class SSHService implements Serializable {
                 logging = LoggingMethod.none
 
                 // Pipe logs to TaskListener's print stream.
-                interaction = {
-                    when(line: _, from: standardOutput) {
-                        logger.println("$remote.name|$it")
+                if(remote.appendName) {
+                    interaction = {
+                        when(line: _, from: standardOutput) {
+                            logger.println("$remote.name|$it")
```

Comment:
can we do something like this instead so we don't repeat as much?

```suggestion
def logPrefix = remote.appendName ? "$remote.name|" : ''

interaction = {
    when(line: _, from: standardOutput) {
        logger.println("$logPrefix$it")
    }
    when(line: _, from: standardError) {
       logger.println("$logPrefix$it")
    }
}
```

Reply:
Sure, let me update it.

Reply:
Updated with https://github.com/jenkinsci/ssh-steps-plugin/pull/12/commits/ce8bccb01d01c7c12749c33d7796b729c699a5fb and also tested by deploying change on a local Jenkins again.